docs/api/features/manifold.py

```diff
@@ -69,19 +64,21 @@ def dataset_example(dataset="occupancy", manifold="all", path="images/", **kwarg
         return
 
     # Create single example
-    _, ax = plt.subplots(figsize=(9, 6))
-    oz = Manifold(ax=ax, manifold=manifold, **kwargs)
 
+    _, ax = plt.subplots(figsize=(9, 6))
```

Comment:
Add an additional line break after this. And adding a comment explaining what the if-else is doing would be great.